var/spack/repos/builtin/packages/py-versioneer/package.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyVersioneer(PythonPackage):
+    """Versioneer is a tool to automatically update version strings (in
+    setup.py and the conventional ‘from PROJECT import _version’ pattern) by
+    asking your version-control system about the current tree."""
+
+    homepage = "https://github.com/warner/python-versioneer"
+    url      = "https://github.com/warner/python-versioneer/archive/0.18.tar.gz"
+    git      = "https://github.com/warner/python-versioneer.git"
+
+    maintainers = ['scemama', 'warner']
```

Comment:
Hi @warner, are you willing to be a maintainer for this Spack package? You don't need to be a Spack expert, and there aren't any real responsibilities that come with being a maintainer. Basically it just gives us someone to ping in case a user has build issues or submits a PR to modify this package.

@scemama make sure to ask for consent before adding maintainers 😄

Reply:
I removed @warner as a maintainer while we wait for the answer.

Reply:
I packaged py-versioneer because it was needed for py-basis-set-exchange.
Now the dependency has been removed (https://github.com/spack/spack/pull/15126#discussion_r384244561), so this package is not used.
I think we should close this pull request and don't merge.

Reply:
You already did all the work to package it, might as well merge it

Reply:
OK, as you prefer :-)